packages/legacy-client/src/client/objectSets/createObjectSetSearchAround.ts

```diff
@@ -25,6 +25,7 @@ import type { SearchAround } from "../interfaces/searchAround";
 import type { OsdkLegacyObjectFrom } from "../OsdkLegacyObject";
 import { createOsdkObjectSet } from "./OsdkObjectSet";
 
+/**@deprecated */
 export function createObjectSetSearchAround<
```

Comment:
I needed to add the links to the OsdkObject type and not just the code gen because the searchAround type we generate here relies on `OsdkLegacyObjectFrom<O,K>`